pkg/proxy/types.go

```diff
@@ -48,23 +48,26 @@ func (spn ServicePortName) String() string {
 type ServicePort interface {
 	// String returns service string.  An example format can be: `IP:Port/Protocol`.
 	String() string
-	// ClusterIP returns service cluster IP.
-	ClusterIP() string
-	// Protocol returns service protocol.
-	Protocol() api.Protocol
-	// HealthCheckNodePort returns service health check node port if present.  If return 0, it means not present.
-	HealthCheckNodePort() int
+	// GetClusterIPString returns service cluster IP in string format.
+	GetClusterIPString() string
```

Comment:
why do we need to rename these?

I feel like the original ones are good enough

Reply:
Yeah I also like the original ones, just that if I don't rename, it can't compile :) As we already have `HealthCheckNodePort` and `Protocol` fields in the struct. This `GetClusterIPString` can be renamed to `ClusterIPString` though.